db/upgrade-jambonz-db.js

```diff
@@ -139,6 +139,11 @@ const sql = {
     'ALTER TABLE lcr_carrier_set_entry ADD FOREIGN KEY lcr_route_sid_idxfk (lcr_route_sid) REFERENCES lcr_routes (lcr_route_sid)',
     'ALTER TABLE lcr_carrier_set_entry ADD FOREIGN KEY voip_carrier_sid_idxfk_3 (voip_carrier_sid) REFERENCES voip_carriers (voip_carrier_sid)',
     'SET FOREIGN_KEY_CHECKS=1',
+  ],
+  8004: [
```

Comment:
please add the record_format alter table statement here so when we upgrade existing databases it is created

Reply:
@xquanluu this is still needed